src/transform.php

```diff
@@ -47,22 +53,12 @@ function callMethod($methodName, ...$args)
  *                              into the array.
  *
  * @return callable
+ *
+ * @deprecated
```

Comment:
Maybe `getElement` and `getProperty` as explicit accessors and `getEntry` as the automatic one you have provided?